src/Microsoft.TemplateSearch.TemplateDiscovery/Program.cs

```diff
@@ -10,55 +12,30 @@ namespace Microsoft.TemplateSearch.TemplateDiscovery
 {
     internal class Program
     {
-        private const bool _defaultRunOnlyOnePage = false;
         private const int _defaultPageSize = 100;
-        private const bool _defaultSaveCandidatePacks = false;
-        private const bool _defaultIncludePreviewPacks = false;
-
-        private const string _basePathFlag = "--basePath";
-        private const string _pageSizeFlag = "--pageSize";
-        private const string _includePreviewPacksFlag = "--allowPreviewPacks";
-        private const string _saveDownloadedPacksFlag = "--savePacks";
-        private const string _runOnlyOnePageFlag = "--onePage";
-        private const string _previousOutputBasePathFlag = "--previousOutput";
-        private const string _noTemplateJsonFilterFlag = "--noTemplateJsonFilter";
-        private const string _verbose = "-v";
-        private const string _providers = "--providers";
-        private const string _packagesPath = "--packagesPath";
 
         private static async Task Main(string[] args)
         {
-            // setup the config with defaults
-            ScraperConfig config = new ScraperConfig()
-            {
-                PageSize = _defaultPageSize,
-                RunOnlyOnePage = _defaultRunOnlyOnePage,
-                SaveCandidatePacks = _defaultSaveCandidatePacks,
-                IncludePreviewPacks = _defaultIncludePreviewPacks
-            };
-
-            if (!TryParseArgs(args, config) || string.IsNullOrEmpty(config.BasePath))
-            {
-                // base path is the only required arg.
-                ShowUsageMessage();
-                return;
-            }
+            RootCommand rootCommand = CreateCommand();
+            rootCommand.Handler = CommandHandler.Create(ExecuteAsync);
```

Comment:
This looks like it will overwrite the handler that was already assigned within `CreateCommand`.